core/src/torii/mod.rs

```diff
@@ -504,9 +504,8 @@ mod tests {
 
             let pagination = Pagination { start, limit };
             handle_queries(state.clone(), pagination, query).map(|result| {
-                if let Scale(QueryResult(Value::Vec(domain))) =
-                    result.expect("Failed request with query")
-                {
+                let Scale(query_result) = result.expect("Failed request with query");
```

Comment:
Is `expect` appropriate? 

When you say `unwrap` I assume that it means "this won't fail". When you say `expect` I assume you mean "it can fail, and if it does, we should panic".

Reply:
This is just a test :). It shouldn't fail

Reply:
well, this is in tests so I would say it can be either way. Ofc, it shouldn't fail if the test is correct, but it could fail if implementation is changed.

Reply:
ok, I'm swapping it with unwrap